src/widgets/wxWidgetsBasicUI.cpp

```diff
@@ -155,7 +155,7 @@ wxWidgetsBasicUI::DoMessageBox(
 }
 
 namespace {
-struct MyProgressDialog : ::ProgressDialog, BasicUI::ProgressDialog {
+struct MyProgressDialog : ::ProgressDialog {
```

Comment:
The first commit can now simply eliminate this class and use `::ProgressDialog` directly.

Reply:
Yeah, makes sense. Surprisingly, MSVC was building as it was, so I've changed this line very late and haven't though that MyProgressDialog is no longer needed at all